pandas/tests/arrays/categorical/test_constructors.py

```diff
@@ -149,6 +149,16 @@ def test_constructor(self):
                          categories=["a", "b", "c", "d"])
         tm.assert_categorical_equal(c1, c2)
 
+        # GH25318
```

Comment:
can you add a 1-liner here explaining what this is; I think that the test_constructor is very huge, would like to have a followup to split this up a bit for different cases.